tba.go

```diff
@@ -227,3 +290,15 @@ func postTbaRequest(resource string, body []byte) (*http.Response, error) {
 	request.Header.Add("X-TBA-Auth-Sig", signature)
 	return client.Do(request)
 }
+
+// Sends a GET request to the TBA API
+func getTbaRequest(path string) (*http.Response, error) {
+  // Make an HTTP GET request with the TBA auth headers
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", fmt.Sprint(tbaBaseUrl, path), nil)
+	if err != nil {
+	  return nil, err
+	}
+	req.Header.Set("X-TBA-App-Id", "cheesy-arena:cheesy-fms:v0.1")
```

Comment:
I looked for a version number yet one wasn't assigned for Cheesy Arena.  It most likely makes sense to have a constant set; however, at this point, there is no versioning system for Cheesy Arena defined.  It may make sense to open a new thread to discuss the proper way to version the software as I feel like that is a larger issue beyond this scope.  Thoughts?